astroquery/desi/tests/test_module.py

```diff
@@ -0,0 +1,132 @@
+import astropy.io.votable
+import pytest
+import os
+import numpy as np
+import pyvo as vo
+
+from astropy.table import Table
+from astropy.io import fits
+from pyvo.dal import TAPResults
+from urllib import parse
+from contextlib import contextmanager
+
+from ... import desi
+from ...utils.mocks import MockResponse
```

Comment:
do use absolute imports, we're slowly cleaning up the codebase, but new stuff should have it.